attribute.tex

```diff
@@ -0,0 +1,5 @@
+\begin{syntax}
+type-qualifier ::= C-type-qualifier | acsl-attribute
```

Comment:
In other grammars like in `ghost.tex`, the rule for type qualifier is `C-type-qualifier ::= C-type-qualifier | ...`. Is there a reason why here the name of the rule is just `type-qualifier`?

Reply:
apart from the fact that I should have checked this point instead of trusting my memory, no reason. I'll fix that, thanks.